.github/workflows/equinix_metal_flow.yml

```diff
@@ -0,0 +1,149 @@
+name: Equinix Metal Action
+
+on:
+  workflow_dispatch:
+
+jobs:
+  Create-runner:
+    name: "Create Runner"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: metal-runner-action
```

Comment:
so logically this is the logic to create a runner which able to replace by self host ec2 runner action/